### DISCUSSION TITLE
how to extend tokenizer for LLaMa-3 ?

### DISCUSSION THREAD

**Discussion by StephennFernandes** [ymcui/Chinese-LLaMA-Alpaca-3#110] [Q&A]
how did you guys extend the tokenizer in LLaMa-3 for chinese ? seems like unlike other llama models in the past this one is based off on a new tiktoken tokenizer. so how did you guys extend the tokenizer ?

**Comment by ymcui** [comment]
We did not extend vocabulary in this repository.

Ref: https://github.com/ymcui/Chinese-LLaMA-Alpaca-3/blob/main/README_EN.md#-uses-the-original-llama-3-vocabulary
